app/assets/stylesheets/components/_card.scss

```diff
@@ -14,7 +14,7 @@
     align-items: center;
   }
 
-  .card-info {
+  &__info {
```

Comment:
should we move that change to the previous PR 🤔 `feature/google-scraping`

Reply:
Rebased. This diff is solved.